iocore/net/quic/QUICStream.cc

```diff
@@ -306,16 +316,14 @@ QUICStream::recv(const std::shared_ptr<const QUICStreamFrame> frame)
   if (this->_recv_offset > frame->offset()) {
     // Do nothing. Just ignore STREAM frame.
     return QUICError(QUICErrorClass::NONE);
-  } else if (this->_recv_offset == frame->offset()) {
```

Comment:
What is the advantage of inserting every frames in `_received_stream_frame_buffer` once?

Reply:
I'd like to leave the received buffer independently. That means we don't need to care about the dup frame, disorder frame or other one. Just push it in buffer and do the logic in received buffer.  Return error code if push failed. Then we can always get the frame in order.

Reply:
Got it. My concern was that when `_received_stream_frame_buffer` become large, looking up the frame could be expensive. But we can optimize it later if we need.

Reply:
That's right !